laze-project.yml

```
@@ -1481,6 +1492,7 @@ modules:
     selects:
       - coap
       - sw/storage
+      - random
```

Because the ordering of these has meaning for laze, I would rather stick to the lexicographical order unless there is a reason not to:

```suggestion
      - random
      - sw/storage
```